exporter/collector/integrationtest/testcases/testcases_metrics.go

```diff
@@ -365,5 +365,36 @@ var MetricsTestCases = []TestCase{
 		// SDK exporter does not support CreateServiceTimeSeries
 		SkipForSDK: true,
 	},
+	{
+		// https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/issues/677
```

Comment:
Skip this test for now.  I'll be adding it in https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/674

Reply:
marked it skip, imo we should still merge it just so it's in the history and you can remove it in your PR